.github/workflows/build.yml

```diff
@@ -3,12 +3,12 @@ name: Continuous integration
 on:
   push:
     paths-ignore:
-      - "CHANGELOG.adoc"
```

Comment:
Why do these double quotes become simple quotes?

Reply:
Because we probably dont have the same style configuration for yml files between members of the team.
I'm going to fix that.